### ISSUE TITLE
Update man page to reflect actual options. I would also like to reduce the # of options available.

### ISSUE THREAD

**Issue by qbit** [qbit/snap#48]

**Comment by evitalis** [comment]
I agree with lowering the number of options available. Were there any in particular you were thinking of removing/combining? I can see `-u/-U` going away when a package/port is available as the OS can handle the updates. I am not sure if `-S` is needed unless the user is running this on a very low powered system as it should be recommended to validate the signatures of the files before installing them.

I am confused by `-s` which says it forces the use of snapshots. Would this be redundant given the nature of the script, and that `VER="snapshots"` is hard coded?

Just my thoughts on the options, not saying any are bad or wrong.

**Comment by qbit** [comment]
Off the top of my head, that was the initial targets for sure. `-s` should have been removed long ago - as it was from when snap "worked" with releases (it never really _worked_, as ABI changes would cause it to die half way through an upgrade).

**Comment by evitalis** [comment]
@qbit Finally getting back around to looking at this. How would you feel about having `-u/-U` combined, and having `-S` removed. I am not sure of a case where someone would not want to check signatures of the downloaded files.
